MANIFEST.in

```diff
@@ -1,2 +1,13 @@
-recursive-include requirements *
+graft requirements
+graft examples
+graft docs
+recursive-exclude docs/generated *
+graft tests
```

Comment:
Why do we want tests in package?

Reply:
https://www.apache.org/legal/release-policy.html#source-packages
```
SOURCE PACKAGES
Every ASF release MUST contain one or more source packages, which MUST be sufficient for a user to build and test the release provided they have access to the appropriate platform and tools. A source release SHOULD not contain compiled code.
```

Reply:
Got it